datamade/committee-oversight
# close out 116 congress

This PR includes a number of client requests, including: 
* added pdf rankings document and link on home page for 116th congress
* showing "projected" appropriately on home page and committee detail page
* adds line for initializing cache table in `docker-compose.db-ops.yml`

## Testing Instructions

 * open the open page and ensure that the link to Download the 116th congress works
 * confirm that the word 'projected' does not appear for the 116th congress on the home page
 * confirm that the word 'projected' does not appear on the committee detail pages
